src/Components/Layout/ManageDataTabs/Chart.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+import { Bar } from "react-chartjs-2";
+
+class Chart extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            chartData: {
+                labels: ['Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange'],
+                datasets: [{
+                    label: '# of Votes',
```

Comment:
looks good, label and data could be a prop in the future (I think you were already planning on doing that tho so)